packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -130,9 +135,17 @@ MeetingInfoUtil.generateOptions = (from) => {
     options.type = _SIP_URI_;
     options.destination = destination;
   }
-  else {
+  else if (MeetingInfoUtil.isConversationUrl(destination, webex)) {
     options.type = _CONVERSATION_URL_;
-    options.destination = `${conversationServiceUrl}/${CONVERSATIONS}/${destination}`;
+    if (destination.includes(`${CONVERSATION_URL_API}${CONVERSATION_URL_GET_API}`)) {
```

Comment:
Do we even need this if/else block? If it is a conversation url, can we just `options.destination = destination;`?

Reply:
no we do not, thats what I was saying above.